e2e/page-objects/header.po.ts

```diff
@@ -1,15 +1,33 @@
 import { browser, element, by } from 'protractor';
 
 export class HeaderPage {
-  userDropdownEl = element(by.css('.navbar-user'));
-  logoutEl = element(by.css('.logout'));
+  megaMenu = element(by.css('.dropdown-lg'));
+  dropdownHeader = element(by.css('.dropdown-header'));
+  notificationsMenu = element(by.css('.bell'));
+  mediaList = element(by.css('.media-list'));
+  languagesMenu = element(by.css('.navbar-language'));
+  languagesList = element(by.css('.languages-menu'));
+  userMenu = element(by.css('.navbar-user'));
+  logout = element(by.css('.logout'));
 
-  clickUserDropdown () {
-    return this.userDropdownEl.click();
+  clickMegaMenu () {
```

Comment:
Rather than having a unique clickAction for each element, it would be more DRY to just get the element and perform whatever action is required.  Either with a generic (parameterized click method) or by simple returning the element to the step and calling whatever method you need there.  Useful since the promise returned by the click can be used to perform further actions in an async fashion.